bblfsh/tree_order.py

```diff
@@ -0,0 +1,12 @@
+class TreeOrder:
```

Comment:
Or data classes, or named tuples, or... :) But will change it on the next commit to `IntEnum` to still allow int comparisons since gRPC will return raw ints.